Add documentation for Mattermost Helm chart (#13728)
gravitational/teleport: docs/pages/includes/plugins/identity-export.mdx

respectively). Later, you will configure the plugin to use these credentials to
connect to the Auth Service directly.

</TabItem>
<TabItem label="Connect to the Proxy or Auth Service" options="Helm Chart">

The following `tctl auth sign` command impersonates the `access-plugin` user,
generates signed credentials, and writes an identity file to the local
directory:

```code
$ tctl auth sign --user=access-plugin --out=auth
```

Then create a Kubernetes secret:

```code
$ kubectl create secret generic teleport-mattermost-identity --from-file=auth_id=auth.pem
```

Teleport's Access Request plugins listen for new and updated Access Requests by
connecting to the Teleport Auth Service's gRPC endpoint over TLS.

The identity file, `auth.pem`, includes both TLS and SSH credentials. Your
Access Request plugin uses the SSH credentials to connect to the Proxy Service,
which establishes a reverse tunnel connection to the Auth Service. The plugin
uses this reverse tunnel, along with your TLS credentials, to connect to the

> Auth Service's gRPC endpoint. The Helm Charts only support this "file" format

Reviewer: ```suggestion
Auth Service's gRPC endpoint. 

The Helm chart only supports the `file` format
```

respectively). Later, you will configure the plugin to use these credentials to
connect to the Auth Service directly.

</TabItem>

> <TabItem label="Connect to the Proxy or Auth Service" options="Helm Chart">

Reviewer: ```suggestion
<TabItem label="Connect to your Teleport Cluster" options="Helm Chart">
```
To make this more general, since in the "Helm Chart" option you'd always be connecting to the Auth Service through the Proxy Service.
